worker/backup_ee.go

```diff
@@ -1,3 +1,4 @@
+//go:build !oss
 // +build !oss
```

Comment:
@matthewmcneely - this change seems to have happened from `auto-fix` code, should I undo this?

Reply:
https://github.com/dgraph-io/dgraph/pull/8191/commits/1161c79cddcdcd5b305085841d56f2f5c0f2f3b7 addressing this here